src/controller/utils.ts

```diff
@@ -160,7 +160,7 @@ export function createTestItem(metaInfo: IJavaTestItem, parent?: TestItem): Test
     }
 
     if (metaInfo.testLevel === TestLevel.Invocation) {
-        item.sortText = metaInfo.id;
+        item.sortText = metaInfo.sortText;
```

Comment:
Let's fallback to `metaInfo.id` when `metaInfo.sortText` is undefined

Reply:
Sounds good! And I'll look into the linting errors asap